packages/core/src/physics/PhysicsScene.ts

```diff
@@ -445,6 +442,16 @@ export class PhysicsScene {
     this._colliders.garbageCollection();
   }
 
+  /**
+   * @internal
+   */
+  _destroy() {
+    if (this._nativePhysicsScene) {
+      this._nativePhysicsScene.destroy();
+      this._nativePhysicsScene = null;
+    }
```

Comment:
Only `this._nativePhysicsScene.destroy();` is OK!

Reply:
done